internalsdk/pro/pro.go

```diff
@@ -77,6 +77,7 @@ func NewClient(baseURL string, userConfig func() common.UserConfig) ProClient {
 			Timeout:   30 * time.Second,
 		}
 	} else {
+		//rt, _ := proxied.ChainedNonPersistent("")
 		httpClient = &http.Client{
 			Transport: proxied.ParallelForIdempotent(),
```

Comment:
I tested with these changes with both `proxied.ChainedNonPersistent` and `proxied. ParallelForIdempotent` http.RoundTrippers